test/parallel/test-tls-alert-handling.js

```diff
@@ -71,15 +70,15 @@ function sendClient() {
 
 
 function sendBADTLSRecord() {
-  var BAD_RECORD = Buffer.from([0xff, 0xff, 0xff, 0xff, 0xff, 0xff]);
-  var socket = net.connect(server.address().port);
-  var client = tls.connect({
+  let BAD_RECORD = Buffer.from([0xff, 0xff, 0xff, 0xff, 0xff, 0xff]);
+  let socket = net.connect(server.address().port);
+  let client = tls.connect({
     socket: socket,
     rejectUnauthorized: false
-  }, function() {
+  }, common.mustCall(function() {
```

Comment:
Interesting, this was a task for the Node Interactive Code & Learn session that I was assigned.  I can remove it if needed.